src/libraries/Native/Unix/System.Native/pal_random.js

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var DotNetEntropyLib = {
+    $DOTNETENTROPY: {
+    },
+    dotnet_browser_entropy : function (buffer, bufferLength) {
+        // check that we have crypto available
+        if (typeof crypto === 'object' && typeof crypto['getRandomValues'] === 'function') {
+            // for modern web browsers
+            // map the work array to the memory buffer passed with the length
+            var wrkArray = new Uint8Array(Module.HEAPU8.buffer, buffer, bufferLength);
+            crypto.getRandomValues(wrkArray);
+            return 0;
+        } else {
+            // we couldn't find a proper implementation, as Math.random() is not suitable
+            abort("no cryptographic support found for getRandomValues. Consider polyfilling it if you want to use something insecure like Math.random(), e.g. put this in a --pre-js: var crypto = { getRandomValues: function(array) { for (var i = 0; i < array.length; i++) array[i] = (Math.random()*256)|0 } };");
```

Comment:
I do not think we should be suggesting an unsecure polyfill in the error message. 

I think we should leave it for the documentation to discuss the polyfill workarounds. The recommended workaround should be something that tries to be secure as much as possible, like the MSR library discussed in the other PR. If somebody wants to use insecure polyfill, I would leave it up to them to figure out how to do it. 

cc @GrabYourPitchforks

Reply:
Changed to let normal managed exception to handle error.

```
WASM-ERR: 
WASM-ERR: Unhandled Exception:
WASM-ERR: System.Security.Cryptography.CryptographicException: Error occurred during a cryptographic operation.
WASM-ERR:    at Interop.GetCryptographicallySecureRandomBytes(Byte* buffer, Int32 length)
WASM-ERR:    at System.Security.Cryptography.RandomNumberGeneratorImplementation.GetBytes(Byte* pbBuffer, Int32 count)
WASM-ERR:    at System.Security.Cryptography.RandomNumberGeneratorImplementation.GetBytes(Span`1 data)
WASM-ERR:    at System.Security.Cryptography.RandomNumberGeneratorImplementation.GetNonZeroBytes(Span`1 data)
WASM-ERR:    at System.Security.Cryptography.RandomNumberGeneratorImplementation.GetNonZeroBytes(Byte[] data)

```

Does this work?